src/Surfnet/StepupRa/RaBundle/Command/SearchRaCandidatesCommand.php

```diff
@@ -22,6 +22,23 @@
 
 class SearchRaCandidatesCommand
 {
+    /**
+     * @Assert\NotBlank(message="ra.search_ra_candidates.actor.blank")
+     * @Assert\Type("string", message="ra.search_ra_candidates.blank.type")
+     *
+     * @var string
+     */
+    public $actorId;
+
```

Comment:
Didn't the MD trigger on the two linefeeds?

Reply:
Apparently it's not a PSR-2 spec to disallow this in class properties.